src/components/Main.tsx

```diff
@@ -225,6 +228,7 @@ const firstSceneScreenOptions: StackNavigationOptions = {
 
 export const Main = () => {
   const theme = useTheme()
+  const [legacyLanding, setLegacyLanding] = React.useState<boolean | undefined>(undefined)
```

Comment:
`undefined` default is redundant here. Also no need for the `| undefined` because it gets inferred.

Reply:
this so setLegacyLanding below works...

Reply:
When I change the type arg to `<boolean>` the `setLegacyLanding` invocation below works just fine. What do you mean?